composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "symplify/symplify",
     "description": "Main repository for Symplify packages development.",
+    "license": "MIT",
```

Comment:
I remember not managing to convince you to keep this in composer.json. What changed your mind?